### PR TITLE
fixed error with setting DateTime property

### PR DESCRIPTION
**Pull request by vtgrady2k**
fixed the following error: System.ArgumentException Invalid Primitive Type System.DateTime. Consider using CodeObjectCreateExpression

### CODE REVIEW

**Comment by rmboggs** [comment]
Hi,

Thank you for this change.  I'm going to run this through to see how it looks.  If it's ok, I'll pull it in.

Thanks,
Ryan

**Comment by vtgrady2k** [comment]
No problem! Nice work on this by the way, this code worked great for generating the document templates I needed for another project I'm working on.

Mike

**Comment by rmboggs** [comment]
Thanks for your kind words.

Looks ok.  Can you do 2 quick things if possible?

- Add a comment on the new section saying something to the effect of DateTime values need to be handled differently than other simple types, just so it is quick to understand what is going on in there.
- Add a note to the change log in the fixed section of the upcoming 0.5.0-beta release.  Feel free to add a note saying "fixed by vtgrady2k" if you like

If not, I can do it but it would help me out if you can put those items in there.

**Comment by vtgrady2k** [comment]
Sure thing, you go it.

File: src/Serialize.OpenXml.CodeGen/OpenXmlElementExtensions.cs [hunk 1]
```diff
@@ -538,10 +538,35 @@ public static class OpenXmlElementExtensions
                 {
                     propVal = val.GetType().GetProperty("Value").GetValue(val);
 
+                    CodeExpression codeExpression;
+
+                    // If the current property value type is a DateTime
```
Comment (rmboggs) [inline src/Serialize.OpenXml.CodeGen/OpenXmlElementExtensions.cs:543]: Very good.  Thanks.

**Comment by rmboggs** [comment]
Perfect, thank you for your help. :)

**Pull request merged**